compiler/rustc_middle/src/query/mod.rs

```diff
@@ -2160,10 +2160,10 @@ rustc_queries! {
         desc { "computing autoderef types for `{}`", goal.value.value }
     }
 
-    query supported_target_features(_: CrateNum) -> &'tcx UnordMap<String, Option<Symbol>> {
+    query known_target_features(_: CrateNum) -> &'tcx UnordMap<String, rustc_target::target_features::Stability> {
```

Comment:
I had interpreted "supported" to mean "supported by this compiler based on stability" but now I'm not even sure if that's correct.  I don't have a strong opinion on the name but I'd be tempted to call this `rust_target_features` to be obvious that this is distinct from any backend/LLVM features

Reply:
Ah makes sense. Yeah I realized it's often not very clear at all right now whether a given feature name is a Rust target feature or an LLVM target feature... at some point we'll probably want to use types to distinguish these, but I'll leave that refactoring to someone else.